elegant/process_experiment.py

```diff
@@ -80,11 +81,23 @@ def segment_experiment(experiment_root, model, channels='bf', use_gpu=True, over
         overwrite_existing: if False, the segmenter will not be run on existing
             mask files, nor will existing annotations be modified even if new
             mask files are generated for a timepoint.
+        timepoint_filter - filter for scan_experiment_dir specifying timepoints to segment
```

Comment:
What's the use-case here? Specifically, how often is it expected to need to re-segment only some images from an experiment?

Reply:
An intended use case is to selectively segment (i.e. the first time) a set of images with a given segmenter. Two already used/forseeable examples:

1. A user trains a classifier that is specifically accurate on a particular life stage (e.g. late larval animals) and only wants to segment those specific images with the classifier.
2. Using the standard adult classifier, the user only wants to segment adult images (to prevent unnecessary runtime lost on segmenting images that don't make sense to segment).

Reply:
gotcha -- makes sense.